### ISSUE TITLE
Incomplete annotations in some examples

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#384]
**Reported by stefanv on 15 Jul 2010 13:58 UTC**
We've discovered some examples where the connections were not displaying in MapleSim. Upon investigation, it turns out that the connections are there but no color is specified. The affected models examples are:

Modelica.Electrical.Analog.Examples.ChauCircuit
Modelica.Electrical.Analog.Examples.DifferenceAmplifier
Modelica.Electrical.Analog.Examples.HeatingNPN_OrGate
Modelica.Electrical.Analog.Examples.NandGate
Modelica.Electrical.Machines.Examples.TransformerTestbench

In each case, the invisible connections should be blue, but the inherited default value is black. Perhaps this bug stems from the fact that color index 0 seems to have mapped to blue in old versions of Dymola?

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/384

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 16 Jul 2010 09:09 UTC**
Fixed in 691c5976b2b1ac504f8c1245086c510cf9ee9522 (blue color introduced in connection lines).

Note, that Modelica 3.2 defines in section 17.5.5.1:

```
record Line
   extends GraphicItem;
   Point points[:];
   Color color = Black;
   ...
end Line;
```

This means that a "Line" is always visible. If no color is defined, the default color "Black" has to be used (which is defined elsewhere in this section).

Therefore, the Electrical.Analog.Examples are correct Modelica models and all connection lines that have no color annotation have to be displayed with "black" color. Still, this is not fully correct from the library point of view, because electrical connection lines should be displayed in blue (which has been corrected).

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 13 Sep 2010 23:21 UTC**
